code-challenges.test.js

```diff
@@ -27,57 +27,53 @@
 
 
 
-describe('arrayShuffler', () => {
-    const colors1 = ["purple", "blue", "green", "yellow", "pink"]
-    const colors2 = ["chartreuse", "indigo", "periwinkle", "ochre", "aquamarine", "saffron"]
-    it("takes in an array, removes the first item from the array and shuffles the remaining content", () => {
-        expect(arrayShuffler(colors1)).toEqual(expect.arrayContaining(["yellow", "blue", "pink", "green"]))
-        expect(arrayShuffler(colors2)).toEqual(expect.arrayContaining(["saffron", "aquamarine", "periwinkle", "indigo", "ochre"]))
-    })
-  })
+// describe('arrayShuffler', () => {
+//     const colors1 = ["purple", "blue", "green", "yellow", "pink"]
+//     const colors2 = ["chartreuse", "indigo", "periwinkle", "ochre", "aquamarine", "saffron"]
+//     it("takes in an array, removes the first item from the array and shuffles the remaining content", () => {
+//         expect(arrayShuffler(colors1)).toEqual(expect.arrayContaining(["yellow", "blue", "pink", "green"]))
+//         expect(arrayShuffler(colors2)).toEqual(expect.arrayContaining(["saffron", "aquamarine", "periwinkle", "indigo", "ochre"]))
+//     })
+//   })
 
 
 // b) Create the function that makes the test pass.
 // Create a function that takes in an array, removes the first item from the array and shuffles the remaining content.
 
 // ***PSEUDOCODE***
-//create a function
+//create a function named arrayShuffler
+//parameter is array
 //remove the first element
 // use .shift method
-//math.floor/math.random
+// use math.floor/math.random method for possible solution
 
 
 
-const colors1 = ["purple", "blue", "green", "yellow", "pink"]
-const colors2 = ["chartreuse", "indigo", "periwinkle", "ochre", "aquamarine", "saffron"]
+//
 
- let arrayShuffler = array  => {
-        array.shift()
-            for (i=0; i < array.length; i++){
-            let random = Math.floor(Math.random() * array.length)
-            return array
-        }
-    }
-console.log(arrayShuffler(colors1))
+//  let arrayShuffler = array  => {
+//         array.shift()
+//             for (i=0; i < array.length; i++){
+//             let random = Math.floor(Math.random() * array.length)
+//             return array
+//         }
+//     }
```

Comment:
While this function will pass your test, it is not necessarily randomizing the result.  Currently the return each time this function is ran the return is:
[ 'blue', 'green', 'yellow', 'pink' ]
[ 'indigo', 'periwinkle', 'ochre', 'aquamarine', 'saffron' ]
The above will pass the test as it has removed the first element, but it has not randomized the remaining elements.  I should see a different variation each time this is ran.